openprescribing/measure_definitions/edoxaban.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Proportion of DOACs prescribed as edoxaban",
+  "title": [
+    "Percentage DOACs as edoxaban"
+  ],
+  "description": [
+    "Percentage DOACs as edoxaban"
+  ],
+  "numerator_short": "Edoxaban items",
+  "denominator_short": "DOACs items",
+  "why_it_matters": [
+    "tbc"
+  ],
+  "tags": [
+    "cardiovascular",
+    "cost"
+  ],
+  "url": null,
+  "is_percentage": true,
+  "is_cost_based": true,
+  "low_is_good": false,
+  "numerator_type": "bnf_items",
+  "numerator_bnf_codes_filter": [
+    "0208020AA # Edoxaban"
```

Comment:
This should be `IS NOT` or similar?

Reply:
this is wrong branch, have created PR for correct branch